scripts/dss_v2_apdapter.py

```diff
@@ -140,13 +144,10 @@ def _parse_staging_area(self) -> Tuple[str, str]:
         Path value will not have a prefix '/' and will have a postfix '/' if not empty.
         """
         split_url = parse.urlsplit(self.args.staging_area)
-        if split_url.scheme != 'gs' or not split_url.netloc:
-            raise ValueError('Staging area must be in gs://<bucket>[/<path>] format')
-        elif split_url.path.endswith('/'):
-            raise ValueError('Staging area URL must not end with a "/"')
-        else:
-            path = f"{split_url.path.lstrip('/')}/" if split_url.path else ''
-            return split_url.netloc, path
+        require(split_url.scheme == 'gs' and split_url.netloc, 'Staging area must be in gs://<bucket>[/<path>] format')
+        reject(split_url.path.endswith('/'), 'Staging area URL must not end with a "/"')
+        path = f"{split_url.path.lstrip('/')}/" if split_url.path else ''
```

Comment:
Use `+` instead of f-string

Reply:
And why lstrip? Didin;t you just exlcude the possibliliity of it ending in slash?

Reply:
`lstrip` because `urlsplit` leaves the slash at the start of the path, and when uploading/copying files to a GSC bucket a prepended slash in the destination file name would cause a folder named '/' to be created.
```
>>> split_url = urllib.parse.urlsplit('gs://domain/path')
>>> split_url
SplitResult(scheme='gs', netloc='domain', path='/path', query='', fragment='')
```

Reply:
I had left and right confused.